services/IoT/applications/api.md

```diff
@@ -42,6 +42,6 @@ Use the [{{site.data.keyword.iot_short_notm}} API V2](https://docs.internetofthi
 
```

Comment:
line 40
"Use the [{{site.data.keyword.iot_short_notm}} API V2]

(https://docs.internetofthings.ibmcloud.com/swagger/v0002.html) to build and customize 

applications for your organization."
Should it be the HTTP API?